meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager_7.0.bbappend

```diff
@@ -2,12 +2,13 @@ FILESEXTRAPATHS_append := ":${THISDIR}/${PN}"
 
 DEPENDS_append = " pulseaudio"
 
-SRCREV = ""
```

Comment:
Is there documentation on why this is required to be set in 2.0? We purposely tracked the tag here